requirements.txt

```diff
@@ -10,7 +10,7 @@ Faker==0.8.11
 future==0.16.0
 hca==4.4.1
 git+git://github.com/hannes-ucsc/jmespath.py@hannes-fix-cache-race#egg=jmespath
-git+git://github.com/HumanCellAtlas/metadata-api@release/1.0b4#egg=hca-metadata-api
+git+git://github.com/HumanCellAtlas/metadata-api@feature/benwu/20-projects-tab-fields#egg=hca-metadata-api
```

Comment:
**TODO:** Change this to release version when release with projects tab changes is made